docs/v0.7/en/greptimecloud/usage-&-billing/wcu-rcu.md

```diff
@@ -0,0 +1,83 @@
+# WCU & RCU
+
+This document will introduce the usage calculation algorithms of GreptimeCloud. To monitor service usage, you can go to the [GreptimeCloud Console](https://console.greptime.cloud/).
+
```

Comment:
Maybe a screenshot here to show where the usage disaplay.

Reply:
The console needs some changes, so the screenshot will be added in another PR in the future.